packages/overture-schema-divisions-theme/src/overture/schema/divisions/division_area/models.py

```diff
@@ -12,10 +12,12 @@
     Named,
     Names,
 )
+from overture.schema.core.ref import RefersTo, Relationship
 from overture.schema.core.types import CountryCode, Id, RegionCode
 from overture.schema.core.validation import (
     exactly_one_of,
 )
+from overture.schema.divisions.division.models import Division
```

Comment:
```suggestion
from ..division.models import Division
```

Reply:
Just updated the PR with a commit that applies this change.